src/Violation/Renderer/HtmlOutput/HtmlOutputRenderer.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SensioLabs\DeprecationDetector\Violation\Renderer\HtmlOutput;
```

Comment:
Namespaces and class names for this and related classes are redundant, and IMO also not correctly nested. We should choose one of:

```
SensioLabs\DeprecationDetector\Renderer\Html\Renderer
[...]
SensioLabs\DeprecationDetector\Renderer\JUnit\Renderer
```

or

```
SensioLabs\DeprecationDetector\Renderer\HtmlRenderer
[...]
SensioLabs\DeprecationDetector\Renderer\JUnitRenderer
```

(my personal preference would be the latter)